src/test/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClientTest.java

```diff
@@ -375,6 +370,33 @@ public void toBulkableAction(){
     assertEquals("{\"doc\":" + idx.payload + ", \"doc_as_upsert\":true}", ba.getData(null));
   }
 
+  @Test
+  public void retryOnConflict() throws Exception {
```

Comment:
is this actually replicating a conflict?

Reply:
No, it's checking for the presence of the `retry_on_conflict` parameter in the request. The retry is done on the server-side by Elasticsearch.